### PR TITLE
Fix screenshot problems with Android API > 24

### PR DESCRIPTION
**Pull request by rafaelje**
This PR works with:

Solve the problem with Fastlane screengrab and API > 24

### CODE REVIEW

**Comment by ajsb85** [comment]
This branch has conflicts that must be resolved

File: .circleci/config.yml [hunk 1]
```diff
@@ -218,6 +253,14 @@ workflows:
             branches:
               only:
                 - develop
+      - screenshot_api_16:
+          requires:
+            - test_api_version_16
+            - test_api_version_25
```
Comment (ajsb85) [inline .circleci/config.yml]: - `test_api_v16`
- `test_api_v25`

File: fastlane/Screengrabfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# remove the leading '#' to uncomment lines
+
+app_package_name 'org.flyve.inventory.agent'
+use_tests_in_packages ['org.flyve.inventory.agent']
+
+app_apk_path 'app/build/outputs/apk/app-debug.apk'
+tests_apk_path 'app/build/outputs/apk/app-debug-androidTest.apk'
+
+locales ['en-US', 'en-GB', 'fr-FR', 'es-ES', 'ko-KR', 'ru-RU']
```
Comment (ajsb85) [inline fastlane/Screengrabfile]: If we download a new version of the Transifex, a new locale is pushed to this array?

File: app/src/androidTest/java/org/flyve/inventory/agent/MainActivityTest.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+public class MainActivityTest {
+    @ClassRule
+    public static final LocaleTestRule localeTestRule = new LocaleTestRule();
+
+    @Rule
+    public ActivityTestRule<Accueil> activityRule = new ActivityTestRule<>(Accueil.class);
+
+    @Test
+    public void InventoryParameters() {
+        if (Build.VERSION.SDK_INT < 24) {
```
Comment (ajsb85) [inline app/src/androidTest/java/org/flyve/inventory/agent/MainActivityTest.java]: The reason is to take only screenshot once? Instead the both tests?

File: .circleci/config.yml (hunk 1 above)
Comment (rafaelje) [reply .circleci/config.yml]: Yes this is the workflows:

- Test API 16 ---\
                            ----- Screenshot API 16
- Test API 25 ---\

Later we can take screenshot for table or other devices

File: fastlane/Screengrabfile (hunk 2 above)
Comment (rafaelje) [reply fastlane/Screengrabfile]: No, need to be manually

File: .gitignore [hunk 4]
```diff
@@ -81,6 +81,7 @@ build
 
 #fastlane
 /fastlane/report.xml
+/fastlane/metadata/
```
Comment (ajsb85) [inline .gitignore]: This folder is generated in the way?

File: app/src/androidTest/java/org/flyve/inventory/agent/MainActivityTest.java (hunk 3 above)
Comment (rafaelje) [reply app/src/androidTest/java/org/flyve/inventory/agent/MainActivityTest.java]: Fastlane Screengrab fail right now with version > 24

File: .gitignore (hunk 4 above)
Comment (rafaelje) [reply .gitignore]: Yes in this folder store the screenshot and all the meta data right now this folder is stored like artifact on CircleCI

File: .circleci/config.yml (hunk 1 above)
Comment (ajsb85) [reply .circleci/config.yml]: Yes, but the idea is to short the names of the labels from `test_api_version_16` to `test_api_v16` for a better view of the graph in the workflow.

File: fastlane/Screengrabfile (hunk 2 above)
Comment (ajsb85) [reply fastlane/Screengrabfile]: ToDo

File: app/src/androidTest/java/org/flyve/inventory/agent/MainActivityTest.java (hunk 3 above)
Comment (ajsb85) [reply app/src/androidTest/java/org/flyve/inventory/agent/MainActivityTest.java]: Ok.

**Pull request merged**